lib/bundle/brew_installer.rb

```diff
@@ -59,11 +59,11 @@ def self.upgradable_formulae
     end
 
     def self.outdated_formulae
-      @outdated_formulae ||= Bundle::BrewDumper.formulae.select { |f| f[:outdated?] }
+      @outdated_formulae ||= Bundle::BrewDumper.formulae.map { |f| f[:name] if f[:outdated?] }.compact
```

Comment:
No problem. This was in lieu of doing .select {g(x)}.map {f(x)}. Is there any even better way than .map {f(x) if g(x)}.compact?